community/rfcs/techniques/proposals/20200205-weight-clustering-proposal.md

```diff
@@ -0,0 +1,82 @@
+# Keras Weight Clustering API
+
+| Status        | Accepted                                      |
+:---------------|:----------------------------------------------|
+| **Author(s)** | Mohamed Nour Abouelseoud (mohamednour.abouelseoud@arm.com), Aron Virginas-Tar (aron.virginas-tar@arm.com), Anton Kachatkou (anton.kachatkou@arm.com)|
+| **Sponsor**   | Alan Chiao (alanchiao@google.com)             |
+| **Updated**   | 2020-03-06                                    |
+
+## Objective
+
+This RFC document addresses the technique &ndash; and consequent API change &ndash; of weight clustering. This is an approach for compressing neural networks by restricting the number of unique weights so that a lower number of bits is required to represent each weight.
+
+## Motivation
+
+Clustering, or weight sharing, allows for certain types of hardware to benefit from advanced weight compression techniques and the associated reduction in model memory footprint and bandwidth.
+Clustering can also be combined with pruning and quantization, reducing the network’s storage requirements even further.
+
+See the [experimental results](#experimental-results) below, obtained across a set of models, that demonstrate the benefits of this technique in terms of weight compression and accuracy preservation.
+
+## Proposed Design
```

Comment:
The proposal's focus is on the benefits and motivation as opposed to the implementation. You can still keep this section, but move it to the bottom.

I'd rename it to "Proposed Approach", to distinguish it from the other RFC with the design.